src/AutorestSwift/View Models/ReturnTypeViewModel.swift

```diff
@@ -31,31 +31,14 @@ import Foundation
 ///     ... -> ReturnTypeName
 struct ReturnTypeViewModel {
     let name: String
-    let statusCodes: [String]
-    /// Identifies the correct snippet to use when rendering the view model
-    let strategy: String
-    let pagingNames: Language.PagingNames?
 
     init(from responses: [ResponseViewModel]?) {
-        var statusCodes = Set<String>()
-        var strategies = Set<ResponseBodyType>()
+        var objectType = Set<String>()
         for response in responses ?? [] {
-            strategies.insert(response.strategy)
-            for statusCode in response.statusCodes {
-                statusCodes.insert(statusCode)
-            }
+            objectType.insert(response.objectType)
         }
 
-        assert(strategies.count <= 1, "Different strategy in ResponseViewModel is currently not supported.")
-        self.statusCodes = Array(statusCodes)
-        self.strategy = strategies.first?.rawValue ?? ResponseBodyType.noBody.rawValue
-
-        let response = responses?.first
-        self.pagingNames = response?.pagingNames
-        if let elementType = response?.pagedElementClassName, response?.pagingNames != nil {
-            self.name = "PagedCollection<\(elementType)>"
-        } else {
-            self.name = response?.objectType ?? "Void"
-        }
+        let objectTypes = Array(objectType)
+        self.name = objectTypes.count > 1 ? "Any" : objectTypes.first ?? "Void"
```

Comment:
Do we have any examples of something returning type `Any`?  Is this a testserver test? I can't imagine in a real-world example this being a good Swift UX.

Reply:
The xms-error-responses.json returns `Any`. For 202 status code, it returns nothing, but for 200 , it returns Pet object.

![image](https://user-images.githubusercontent.com/8701731/93143086-15d78b80-f69c-11ea-902f-4cd0cc686cde.png)

Reply:
Perhaps then, it makes more sense to return `Pet?`

Reply:
The scenario would need to be a non-error response of `.noBody` and exactly one `.body` response. Otherwise, I think we should throw an assertion failure.

Reply:
Thanks for the suggestion. Changed.